src/Components/Theme.js

```diff
@@ -1,5 +1,20 @@
 
 const Theme = {
+  components: {
+    MuiButton: {
+      variants: [
+        {
+          props: { size: 'square' },
```

Comment:
Adding the style to the theme.js is a nice way to add reusability if we were to use it again.
However, abstracting before we need to adds complexity without any benefit.
As a rule of thumb, unless there's code repeated three times it's usually better not abstracted away unless it's obvious you'll be using that again.
We don't really care too much about maintainability(as it's doubtful anyone will come back to this) so will approve, but note for next time~